src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidgetUser/DetailsWidgetUser.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import TextDecorator from '~lib/TextDecorator';
+import UserMention from '~core/UserMention';
+import { Address } from '~types/index';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import MaskedAddress from '~core/MaskedAddress';
+import styles from './DetailsWidgetUser.css';
+
+const displayName = 'dashboard.ActionsPage.DetailsWidget.DetailsWidgetUser';
+
+interface Props {
+  username?: string;
+  walletAddress: Address;
+}
+
+const DetailsWidgetUser = ({ walletAddress, username }: Props) => {
+  const { Decorate } = new TextDecorator({
+    username: (usernameWithAtSign) => (
+      <UserMention username={usernameWithAtSign.slice(1)} />
+    ),
+  });
+  const UserAvatar = HookedUserAvatar({ fetchUser: false });
+
+  return (
+    <div className={styles.main}>
+      <UserAvatar size="s" address={walletAddress || ''} />
+      {username && (
+        <div className={styles.username}>
+          <Decorate key={walletAddress}>{`@${username}`}</Decorate>
```

Comment:
Please don't use decorate in here. 

Decorate is for places where you don't know if a username is going to exist in a given piece of text _(Eg: comments)_. Eg:
```
Hello @user who are you?
```
Will decorate every instance of a username it finds in the above text. But that means parsing it all.

In places like this, where you know you're going to display a username it's pointless, as you can more easily achieve this by just styling the username yourself.

Please remove `Decorate` in this instance